modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,78 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.5 - 2023-06-01
+
+=== Added
+* New `table_merge_content_on_paste` option which disables the merging behaviour when pasting a table inside an existing table.
+* New optional `defaultExpandedIds` and `onToggleExpand` options to the `tree` component config.
+* New optional `defaultSelectedId` option to the `tree` component config.
+* New `help_accessibility` option which displays the keyboard shortcut to access the help functionality in the statusbar.
```

Comment:
```suggestion
```

Reply:
This has been moved to `6.6` @brian-forte

Reply:
To quote my commit message:

> Removed an entry from Improved section of 6.5-specific changes:
> 
> * New `help_accessibility` option which displays the keyboard shortcut to access the help functionality in the statusbar.
> 
> This change has been moved to the 6.6 release.